share/lib/python/neuron/tests/test_neuron.py

```diff
@@ -194,6 +194,8 @@ def testRxDexistence(self):
 
     def test_newobj_err(self):
         '''Test deletion of incompletely constructed objects'''
+        # TODO : temporary disable for testing
```

Comment:
There is no need to disable this test. It passes. The test that fails is the next one.

(It's a little confusing because this test makes sure error handling doesn't break things, but you'll note that after the error message you get a message that there are correctly 0 objects and the "ok")